query-node/schemas/council.graphql

```diff
@@ -0,0 +1,220 @@
+# TODO:
+# - do we need some fulltext search for council/election?
+
+# workaround for https://github.com/Joystream/hydra/issues/434
+type VariantNone @variant {
+  _phantom: Int
+}
+
+################### Council ####################################################
+
+type CouncilStageUpdate @entity {
+  "The new stage council got into."
+  stage: CouncilStage!
+
+  "Block number at which change happened."
+  changedAt: BigInt!
+
+  "Council elected in this stage update if any."
+  electedCouncil: ElectedCouncil
+}
+
+type CouncilStageAnnouncing @variant {
+  "Number of candidates aspiring to be elected as council members."
+  candidatesCount: BigInt!
+}
+
+type CouncilStageElection @variant {
+  "Number of candidates aspiring to be elected as council members."
+  candidatesCount: BigInt!
+}
+
+type CouncilStageIdle @variant {
+  # no properties
+
+  # TODO: remove me - variant needs to have at least 1 property now
+  dummy: Int
+}
+
+union CouncilStage = CouncilStageAnnouncing | CouncilStageElection | CouncilStageIdle | VariantNone
+
+type Candidate @entity {
+  "Account used for staking currency needed for the candidacy."
+  stakingAccountId: String!
+
+  "Account that will receive rewards if candidate's elected to the council."
+  rewardAccountId: String!
+
+  "Election cycle"
+  cycleId: ElectionRound!
+  stake: BigInt!
+  votePower: BigInt!
+  note: String!
```

Comment:
@ondratra I think that we also need the `Note` [metadata](https://joystream.gitbook.io/joystream-handbook/key-concepts/encodings#council-candidacy-note) to be expended and included in the GraphQL mappings – as other metadata is stored in the query-node. For instance, the [membership metadata](https://joystream.gitbook.io/joystream-handbook/key-concepts/encodings#membership-metadata) is available on the [`Member` entity](https://github.com/ondratra/joystream/blob/fb2d6b604bb98c338db415e65627e9baae6d9f24/query-node/schemas/membership.graphql#L38-L39).